### PR TITLE
Add Vite Support

### PR DESCRIPTION
**Pull request by GregLahaye**
Adds support for Vite, closes https://github.com/pixijs/storybook/issues/1.

Created packages for `@pixi/storybook-preset-vite` and `@pixi/storybook-vite` by copying the Webpack ones and modifying them to use Vite.

I've created an example project using this framework here: https://github.com/GregLahaye/pixi-vite-storybook-example.

image

image

### CODE REVIEW

**Review by baseten — COMMENTED** [review]
Thanks so much for your contribution!

I've got one small query regarding the Vite config

File: packages/storybook-preset-vite/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import type { StorybookConfig } from "./types";
+
+export * from "./types";
+
+export const vite: StorybookConfig["viteFinal"] = (config) => {
+  return config;
+};
```
[the remark this answers is not in the document]
Comment (GregLahaye) [reply packages/storybook-preset-vite/src/index.ts:7]: Yep! Vite will handle all the assets without extra config.
Feel free to reach out if there any other questions/changes you want for this PR, cheers!

**Comment by baseten** [comment]
@GregLahaye sorry for the slow response around this, are you able to fix the `package-lock.json` conflict and I can get it merged? Please make sure you're using the same node/npm version as the repo is expecting, which is `v16.17.0`

**Comment by GregVGW** [comment]
@baseten Sorry for the delay on this, there were some changes to APIs so I've created a new PR here: https://github.com/pixijs/storybook/pull/14

**Pull request closed without merging**